types/meteor/index.d.ts

```diff
@@ -2145,3 +2145,67 @@ declare module "meteor/accounts-base" {
         function onLogout(func: (user: Meteor.User, connection: Meteor.Connection) => void): void;
     }
 }
+
+declare module "meteor/webapp" {
```

Comment:
But other similar built-in meteor modules was declared in same way, every module has to be seperated then, right?